src/Math-Tests-Random/PMBinomialGeneratorTest.class.st

```diff
@@ -56,7 +48,7 @@ PMBinomialGeneratorTest >> testSampleMeanConvergesToExpectedValue [
 		numberOfTrials: numberOfTrials
 		probabilityOfSuccess: probabilityOfSuccess.
 	
-	sample := self
+	sample := SampleNumericalData
```

Comment:
Using a helper class is fine with me, but since it's in PolyMath it should have the PM prefix in its name. I'd also prefer to have "random" somewhere in the same, as "sampling" is also used for deterministic approaches (e.g. sampling a continues signal to obtain a discrete time series). Perhaps "PMRandomSampling" or something similar?

(Warning: I am notorious for changing my mind about names frequently. I might dislike my own proposition tomorrow.)

Reply:
I forgot the PM prefix! I'll make that change.

I think given the context is the random package I'm OK with sampling. With reference to deterministic processes, the meaning seems to be different in that context, much like weight in physics is different from (statistical) weight.

Reply:
Then the code would like like `PMRandomSample ofSize:1000 usingGenerator: generator`
which I certainly think reads well, @khinsen